src/Workspaces/Core/Portable/project.json

```diff
@@ -1,8 +1,24 @@
 ﻿{
-  "dependencies": {
-    "Newtonsoft.Json": "6.0.4"
-  },
-  "frameworks": {
-    ".NETPortable,Version=v4.5,Profile=Profile7": {}
-  }
+    "supports": {
+        "net46.app": {}
+    },
+    "dependencies": {
+        "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
+        "Newtonsoft.Json": "6.0.4",
+        "System.Threading.Tasks": "4.0.11",
```

Comment:
Hrmm... whne i switched to that, i got these errors:

Utilities\CompilerUtilities\ImmutableHashMap.cs(13,47): error CS1069: The type name 'Contract' could not be found in th
e namespace 'System.Diagnostics.Contracts'. This type has been forwarded to assembly 'System.Diagnostics.Contracts, Ver
sion=0.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' Consider adding a reference to that assembly. [C:\GitHu
b\roslyn-internal\Open\src\Workspaces\Core\Portable\Workspaces.csproj]
Workspace\Workspace.cs(881,39): error CS0234: The type or namespace name 'PureAttribute'